frontend-shared/package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "@hypothesis/frontend-shared",
+  "version": "1.0.0",
```

Comment:
Similar to the root client package, we probably want to figure out a version paradigm here too controlled by jenkins or github actions.